test/unit/helpers/ConfigHelper.ts

```diff
@@ -147,8 +146,15 @@ test(`promptOptions 6 - autoRegister = true backwards compatibility for custom i
   };
   (fakeUserConfig as any).promptOptions = {
     slidedown: {
-      enabled: true,
-      autoPrompt: false,
+      prompts: [{
```

Comment:
Should keep the old format so we are still testing what could be in the wild as is.

Can use `getPromptOptionsFromFinalAppConfig` to make the tests pass.